tasks/src/audio-to-audio/about.md

```diff
@@ -2,7 +2,7 @@
 
 ### Speech Enhancement (Noise removal)
 
-Speech Enhancement is a bit self explanatory. It improves (or enhances) the quality of an audio by removing noise. There are multiple libraries to solve this task, such as Speechbrain, Asteroid and ESPNet. Here is a simple example using Speechbrain 
+Speech Enhancement is self explanatory. It improves (or enhances) the quality of an audio by removing noise. There are multiple libraries to solve this task, such as Speechbrain, Asteroid and ESPNet. Here is a simple example using Speechbrain
```

Comment:
I'd rather keep "a bit" :D it's not that self explanatory. We can keep the language playful in these pages, it doesn't have to be very formal